Algorithms.Tests/Numeric/FactorialTests.cs

```diff
@@ -38,5 +39,60 @@ public static void GetsFactorial(int input, long expected)
 
             _ = Assert.Throws<ArgumentException>(Act);
         }
+
+        // Reference for factorial values are taken from https://coolconversion.com/math/factorial/
+        [Test]
```

Comment:
`Test` attribute is redundant if there's a `TestCase` attribute specified

Reply:
Issue resolved in new pull request